templates/solutions/secure-open-source/index.html

```diff
@@ -0,0 +1,767 @@
+          <p>Landscape reduces the cost of managing open source:</p>
+          <hr class="p-rule is-muted" />
+          <p>
+            <a href="https://ubuntu.com/kubernetes">Discover Canonical Kubernetes&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+
+      <hr class="p-rule is-muted" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/dd859f20-Canonical Product Logo__OpenStack.png",
+              alt="Canonical OpenStack",
+              width="810",
+              height="121",
+              hi_def=True,
+              loading="lazy",
+              attrs={"style": "max-width: 250px;"}
+              ) | safe
+            }}
+          </h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>Build your enterprise private cloud with Canonical OpenStack. Get 10 years of security updates, cloud hardening options and meet your compliance requirements.</p>
+          <hr class="p-rule is-muted" />
+          <p>
+            <a href="https://ubuntu.com/openstack/features">Read about Canonical OpenStack&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>For resilient and compliant IoT device deployments</h2>
+    </div>
+    <div class="col">
+      <p>
+        With Ubuntu Pro all your software is security-maintained from the moment you install it, no matter where you install it. 
+      </p>
+      <p>Ubuntu Pro for Devices simplifies security and compliance for IoT device deployments, providing 10 years of peace of mind for Ubuntu and thousands of open source packages, including popular robotics frameworks like ROS.</p>
+      <p>Pro for Devices can be combined with <a href="https://ubuntu.com/certified">Ubuntu Certified Hardware</a> to offer a best in class Ubuntu experience on devices out-of-the-box.</p>
+      <hr class="is-muted" />
+      <a href="https://canonical.com/blog/ubuntu-pro-for-devices">Learn more about Ubuntu Pro for Devices&nbsp;&rsaquo;</a>
+    </div>
+  </div>
+</section>
+
+<hr class="p-rule is-fixed-width" />
+
+<section class="p-strip is-deep">
+  <div class="u-fixed-width">
+    <h2>
+      Get a decade of peace of mind.<br />
+      <a href="/contact-us" aria-controls="secure-open-source-modal">Contact our team&nbsp;&rsaquo;</a>
+    </h2>
+  </div>
+</section>
+
+<section class="p-section--deep">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row--50-50">
+    <div class="col">
+      <h2>Build a secure open source supply chain</h2>
```

Comment:
Sorry forgot to flag yesterday, but this should have a break after "source" to match design 
![image](https://github.com/canonical/canonical.com/assets/17607612/18a8d777-9e5e-4909-b80b-a8803770d11a)